strr-platform-web/app/stores/terms-of-service.ts

```diff
@@ -0,0 +1,63 @@
+interface TOSPatchResponse {
+  isTermsOfUseAccepted: boolean
+  termsOfUseAcceptedVersion: string
+}
+
+interface TOSGetResponse {
+  isTermsOfUseAccepted: boolean
+  termsOfUseAcceptedVersion: string | null
+  termsOfUseCurrentVersion: string
+  termsOfUse: string
+}
+
+// TODO: make this generic for the core layer?
```

Comment:
Yeah I think this would be good there. Might be good to prefix this with 'connect' if we're going to do that. For now we could put it in the strr base layer and figure out the kinks before moving it